Resources/doc/reference/page_composer.rst

```diff
@@ -79,6 +79,21 @@ Configure
                   B: content_bottom
                   F: footer
 
+
+Template inheritance
+^^^^^^^^^^^^^^^^^^^^
+
+If you want your template to extend from another template so you don't
+have to redeclare all your containers you can use the ``inherits_containers`` option.
+This is also shown in the example above.
+
+.. code-block:: yaml
+
+      templates:
+           2columns:
+              inherits_containers: default
```

Comment:
Can you add some other containers to demonstrate, that you can use inheritance AND new container.